server/controllers/auth/userAuth.js

```diff
@@ -114,5 +114,99 @@ class AuthController {
       return next(err);
     }
   }
+
+    /**
+   *@description Send Reset password Email to the user
+   * @param {object} req request to the sent
+   * @param {object} res respond gotten form server
+   * @param {object} next  callback funtion
+   * @returns {object} an object when base on if successful or not
+   */
+  static async sendResetEmail(req, res, next) {
+    const { email } = req.body;
+    try{
+      const user = await User.findOne({
+          where: {
+            email
+          },
+          attributes: {
+            exclude:
+            [
+              'createdAt',
+              'updatedAt',
+              'password'
+            ]
+          }
+      });
+      const resetDuration = '30m';
+      const { host } = req.headers;
+      const { dataValues } = user;
```

Comment:
what are we expecting as the value of "dataValues"?

Reply:
"dataValues" contains the user data object after the search(such as id, email and name). The user data is needed to generate a token